src/components/navbar/navbar.tsx

```diff
@@ -1,59 +1,121 @@
+"use client";
 import Link from "next/link";
 import "./navbar.css";
 import logoOakyardOnDark from "../../../public/photos/logos/Logo_Oakyard_on_dark.svg";
 import Image from "next/image";
+import { AiOutlineMenu } from "react-icons/ai";
+import { AiOutlineClose } from "react-icons/ai";
+import { useState } from "react";
+
+const navlist = [
+  {
+    navName: "For Talents",
+    to: "#",
```

Comment:
@busratok i see there is too much gap between `For talents`  and the `oakyard` logo. Can we reduce it sensibly? To me it looks kind of disjoint from the rest

Reply:
Thank you for bringing this to my attention. I fixed it.

Reply:
@busratok i still see some gap, for me say it should be an even distance both from the logo and the support us button.
Also, as you can see at the bottom, there are conflicts to solve

Reply:
In the design file, there is no even distance. I put the gap according to it.

![navbar-gap](https://github.com/bisonaide-org/whakaako-website-react/assets/119109194/455bebab-f8ff-4a25-a2d2-4eabe272b15d)

Reply:
ok it wasn't just looking good to me. We can change it later on if we want thanks @busratok